orbit/pkg/update/update.go

```diff
@@ -34,9 +34,21 @@ import (
+// Defining these as variables so that we can overwrite them during development/testing of the migration.
+var (
```

Comment:
Should these be changed back to `const` now that testing is completed?

Reply:
I've created https://github.com/fleetdm/fleet/issues/24519 to do this before releasing, because the QA team will need some of the tooling to test this locally.